.github/workflows/dbml.yaml

```diff
@@ -0,0 +1,24 @@
+name: Build database documentation
+
+on:
+  push:
+    paths:
+      - "database.dbml"
```

Comment:
Will it be better if only update when `main` branch pushes?

Reply:
As stated in the PR description, there is currently no database documentation specifically for the production environment.

For development and discussion, it is still desirable to keep the information on dbdocs as up-to-date as possible.